enterprise/backend/src/metabase_enterprise/data_editing_public/api.clj

```diff
@@ -23,7 +23,8 @@
     (api/check-400 (seq rows) "Please supply at least one row.")
     (api/check-400 (every? seq rows) "Every row should not be empty.")
     (binding [api/*current-user-id* (:creator_id hook)]
-      (data-editing/insert! (:table_id hook) rows))))
+      (data-editing/insert! (:table_id hook) rows))
+    {:created (count rows)}))
```

Comment:
But it should be `(count (:created_rows (insert! ...)))`
Otherwise we're lying. 😁

Reply:
When can they be different though? 🤔

Reply:
I mean your code is definitely nicer and I'll make the change, just curious about the edge cases you can think of.

Reply:
The edge case is that upcoming changes will cargo-cult your behavior and I think this shortcut will be revered in particular since it's obvious you could do the other thing, so probably you had your reasons? :)